packages/client/src/components/lemon-logo.svg

```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 114.87 76.59" height="38">
```

Comment:
This should probably be moved to assets. And maybe cleaned up a bit:

https://fvsch.com/svg-icons

Reply:
Done, thank you! 
Logo svg's are being used in components, react would not allow import from outside src dir, so I moved assets to src (prev at top level in project).  Is this bad practice for any reason, or breaking precedence?